lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -332,7 +332,8 @@ class EngineAutofillForm {
 
```

Comment:
I think the issue in Safari is caused by this code removing and re-adding the form (see `form?.remove();` right above this.

Since the line wasn't changed, it's probably something *else*, but I still haven't figured out how this whole system works.

Reply:
(The problem was `handleBlur` in safari moving focus after blur.)